desktop/src/com/unciv/app/desktop/DesktopLauncher.kt

```diff
@@ -32,6 +32,8 @@ internal object DesktopLauncher {
 
     @JvmStatic
     fun main(arg: Array<String>) {
+        
+        System.setProperty("org.lwjgl.opengl.Display.allowSoftwareOpenGL", "true")
```

Comment:
SOME devices can not run Unciv because software OpenGL mode is not allowed. 
Put this code `System.setProperty("org.lwjgl.opengl.Display.allowSoftwareOpenGL", "true");`
The problem will be solved. This will allow libgdx to run as software openGL mode.

Reply:
Yes, I'm asking which devices. Is this a software problem, certain OS's? A hardware one?

Reply:
I think this is a hardware problem.Libgdx requires at least OpenGL 1.5 on the desktop.But SOME people don't know how to update the OpenGL.So I suggest putting the code.

Reply:
1st generations Intel HD Graphics running under Windows 10 with Java 8u60 and newer. https://github.com/LWJGL/lwjgl/issues/119
Downgrading java to 8u51 solved problem for few people in discord.
This exception also can be caused by outdated or incompatible driver. 
There was 1 person in discord with this problem recently. He had intel B960 and windows updated drivers to 9.17.10.4459 (Windows 10 does not have official GPU support for Sandy Bridge); downgrading drivers to 4229 - latest compatible with his intel GPU - solved the problem